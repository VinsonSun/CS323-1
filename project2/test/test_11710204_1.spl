int test_11710204_1()
{
  int a = 3;
  a = a + b;
  return a;
}
