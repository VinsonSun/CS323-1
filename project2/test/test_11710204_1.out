Error type 1 at Line 4: undefined variable: b
